### PR TITLE
Show glossary options on focus.

### PR DESCRIPTION
**Pull request by jmcarp**
* Upgrade to latest typeahead.js.
* Show default glossary options on focus.

NB: Still needs work on updating styles for latest typeahead.js.

@noahmanger: Help with styles much appreciated!

### CODE REVIEW

File: static/styles/_components/_search-bar.scss [hunk 1]
```diff
@@ -56,36 +56,33 @@
...
-  display: block;
-  color: #fff;
-  padding: 5px 20px;
-  background: $dark-gray;
-}
-
-.tt-suggestions {
-  color: #000;
-}
+// .tt-menu:hover {
```
Comment (msecret) [inline static/styles/_components/_search-bar.scss]: Is there a reason this is commented out rather then removed?

File: static/styles/_components/_search-bar.scss [hunk 2]
```diff
@@ -56,36 +56,33 @@
...
+//   display: block;
+//   color: #fff;
+//   padding: 5px 20px;
+//   background: $dark-gray;
+// }
 
 .tt-suggestion {
-  @include clearfix();
+  // @include clearfix();
```
Comment (msecret) [inline static/styles/_components/_search-bar.scss]: Same here

**Comment by jmcarp** [comment]
@msecret: This is an unfinished patch (JS updates should be good to go, but styles still need work). It sounds like we might stop using typeaheads for the glossary, but if we stick with them, I was hoping to get some help with styling from you or @noahmanger.

**Comment by jmcarp** [comment]
@noahmanger: What are your thoughts about typeahead compared to something like list.js for the glossary? If we're going to stick with typeahead, showing the first N results and filtering down as the user types, I'll get this patch ready to merge. If we're going to use a different solution, I think we can close this PR.

**Comment by noahmanger** [comment]
Personally, I think I'd rather use Listjs (or something comparable), and
show _all_ results by default in a way that people can scroll through, or
start typing to filter down. Seems like a more natural interaction model
(filtering, not searching), especially with such a finite list of terms.
But open to arguments to the contrary.

On Tue, Jun 2, 2015 at 8:58 AM, Joshua Carp <notifications@github.com>
wrote:

> @noahmanger <https://github.com/noahmanger>: What are your thoughts about
> typeahead compared to something like list.js for the glossary? If we're
> going to stick with typeahead, showing the first N results and filtering
> down as the user types, I'll get this patch ready to merge. If we're going
> to use a different solution, I think we can close this PR.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/18F/openFEC-web-app/pull/215#issuecomment-108000191>.
>



-- 
Noah Manger
18F <http://18f.gsa.gov> | Work: 415-696-6146

**Comment by jmcarp** [comment]
Sounds good to me. Closing this for now. We may still want to upgrade typeahead versions down the line, but I think that can be deferred.

**Pull request closed without merging**